libs/extensions/angular/sidebar-menu/src/components/menu-item/menu-item.component.scss

```
@@ -0,0 +1,104 @@
+@use '@kirbydesign/core/src/scss/utils' as utils;
+
+:host {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 1;
+  align-items: center;
+  align-self: stretch;
+  box-sizing: border-box;
+  min-height: 32px;
+  border-radius: var(--kirby-border-radius-s, 8px);
+  padding: var(--kirby-spacing-xxxs, 4px) var(--kirby-spacing-s, 16px);
+  gap: var(--kirby-spacing-xxs, 8px);
+  border: none;
+  background: inherit;
+  color: inherit;
+  text-decoration: none;
+  cursor: pointer;
+
+  &.lg {
+    min-height: 40px;
```

In place of better support for sizes in our CSS custom props (only spacings currently), I think it is fine to use the `utils.size();` for any size related stuff.
```suggestion
    min-height: utils.size('xl');
```

---

have changed this and most other sizes to use `utils.size()`